tests/meltano/cloud/cli/test_project.py

```diff
@@ -391,3 +391,36 @@ def test_project_use_by_name_and_id_error(self, config: MeltanoCloudConfig):
         )
         assert result.exit_code == 2
         assert "The '--name' and '--id' options are mutually exclusive" in result.output
+
+    def test_project_create(
+        self,
+        projects: list[CloudProject],
+        httpserver: HTTPServer,
+        config: MeltanoCloudConfig,
+    ):
+        for project in projects[:3]:
+            httpserver.expect_oneshot_request(
+                f"/projects/v1/{project['tenant_resource_key']}",
+                method="POST",
+            ).respond_with_json(project)
+            result = CliRunner().invoke(
+                cli,
+                (
+                    "--config-path",
+                    config.config_path,
+                    "project",
+                    "create",
+                    "--name",
+                    project["project_name"],
+                    "--repo-url",
+                    project["git_repository"],
+                    "--root-path",
+                    project["project_root_path"],
+                ),
+            )
+            assert result.exit_code == 0, result.output
+            assert result.output == (
+                f"Project {project['project_name']!r} created successfully.\n"
+                + json.dumps(project)
+                + "\n"
+            )
```

Comment:
```suggestion
            expected = (
                "\r\r\r⠋ Creating project - this may take several minutes...\r"
                "                                                      "
                f"\rProject {project['project_name']!r} created successfully.\n"
            )
            assert result.output == expected
```